build.sbt

```diff
@@ -90,14 +74,11 @@ ThisBuild / testFrameworks += new TestFramework("munit.Framework")
 
 lazy val root = project
   .in(file("."))
-  .settings(scalaVersion := "2.13.3")
```

Comment:
I believe build is failing because this is settings required to cross-compile correctly with the aggregated projects.